app/views/hyrax/my/collections/index.html.erb

```diff
@@ -92,3 +92,6 @@
     </div>
   </div>
 </div>
+
+<!-- Render Modals -->
+<%= render 'modal_add_collection_to_collection' %>
```

Comment:
@jeremyf Is there a test to see if any? collection type is nestable?  That way the modal only gets rendered if there is a possibility that nesting might occur.